api/tests/opentrons/protocol_engine/clients/test_sync_client.py

```diff
@@ -327,3 +327,37 @@ def test_thermocycler_deactivate_lid(
     result = subject.thermocycler_deactivate_lid(module_id="module-id")
 
     assert result == response
+
+
+def test_blow_out(
+    decoy: Decoy,
+    transport: AbstractSyncTransport,
+    subject: SyncClient,
+) -> None:
+    """It should execute a blow_out command."""
+    request = commands.BlowOutCreate(
+        params=commands.BlowOutParams(
+            pipetteId="123",
+            labwareId="456",
+            wellName="A2",
+            wellLocation=WellLocation(
+                origin=WellOrigin.BOTTOM,
+                offset=WellOffset(x=0, y=0, z=1),
+            ),
```

Comment:
Same comment as https://github.com/Opentrons/opentrons/pull/10678/files#r893943417: try not to use data for value objects in tests that looks like it might be valid/used if it's not part of the test.

Here, I would simply use `WellLocation`

```suggestion
            wellLocation=WellLocation(),
```